src/checkers/lines.rs

```diff
@@ -72,6 +73,12 @@ pub fn check_lines(
             }
         }
 
+        if enforce_mixed_spaces_and_tabs {
+            if let Some(diagnostic) = mixed_spaces_and_tabs(index, line) {
+                diagnostics.push(diagnostic);
```

Comment:
By using the "physical" lines here, this will also flag cases like: a multi-line string, where one of the lines in the string contains a mix of tabs and spaces.

But, it looks like `pycodestyle` flags those too, and it seems rare (maybe even intended / helpful), so all good.

Reply:
I used leading_space in the rule, doesn't that just get the "indent"?

Reply:
Yeah totally. What I'm referring to though is something like this:

```py
foo = """
<imagine this is a tab><imagine this is a space>Some content
"""
```

`lines.rs` just iterates over the lines of the program, so it'd first see `foo = """`, then the line that has a tab, space, and "Some content", then a line of `"""`.

So that second line would trigger this check, even though it's kind of _not_ indentation, it's in the body of a string.

IMO it'd be "more correct" _not_ to flag that, but there's not an obvious right decision, and the behavior here is consistent with `pycodestyle`.